src/rsuite/SelectUI.tsx

```diff
@@ -273,26 +275,26 @@ export const SelectPopupUI = observer(function SelectPopupUI_<T>(p: { s: AutoCom
                     <li
                         key={index}
                         style={{ minWidth: '10rem' }}
-                        className={`p-0.5 hover:bg-base-300 cursor-pointer ${index === s.selectedIndex ? 'bg-base-300' : ''}`}
-                        tw={['flex items-center gap-1']}
+                        className={`active:bg-base-300 cursor-pointer ${index === s.selectedIndex ? 'bg-base-300' : ''}`}
```

Comment:
nitpick: would probably be slightly better in the `tw={[..., ...]}` below, possibly using some of the conditional class syntax added with tw